README.md

```diff
@@ -1,3 +1,7 @@
+To contribute to Backstage, base your changes to the main branch: https://github.com/simplybusiness/backstage-source/tree/main
+
+The master branch has changes that are relevant internally at Simply Business which means it is incompatible with the upstream Backstage repository.
```

Comment:
Not fully following what are the changes that master. Why can't we update the master branch to latest code?

Reply:
These are the changes in master which is why we cannot merge master to upstream nor can we base any branches we  want to merge upstream on this master: https://github.com/backstage/backstage/compare/master...simplybusiness:master

Reply:
Make sense